.github/workflows/comment-changes.yml

```diff
@@ -0,0 +1,49 @@
+name: comment-changes
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  aeruginous:
+    if: |
+      !contains(github.event.head_commit.message, '[Aeruginous]') && \
+      !contains(github.event.head_commit.message, 'Chore:') && \
+      !contains(github.event.head_commit.message, 'Docs:')
```

Comment:
It did not work as it should; this workflow should have created a fragment for the PR introducing it.  I assume that these lines are the problem.  I thought that splitting the condition into multiple lines would work but this is most likely to be the reason for the failing workflow run.  I will fix it tomorrow by putting all condition parts into one line.

I am sorry.

Reply:
No problem and take your time adjusting it. Good software never work from the first run :smile: